### PR TITLE
Implementation of Sqrt without using math library

### PR DESCRIPTION
**Pull request by jalajdorai**
Fixes #2214 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Implementation of Sqrt without using the math library

#### Languages Used:
- C++

#### Files Added:
- Implent Sqrt(x) without using the math library.txt

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Sqrt/Implement Sqrt(x) without math library.txt [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/*Implement int sqrt(int x).
+
+Compute and return the square root of x, where x is guaranteed to be a non-negative integer.
+
+Since the return type is an integer, the decimal digits are truncated and only the integer part of the result is returned.*/
```
Comment (SakshiShreya) [inline Sqrt/Implement Sqrt(x) without math library.txt]: Split this line. One line can contain max 80 characters.

File: Sqrt/Implement Sqrt(x) without math library.txt [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+class Solution {
+public:
+    int mySqrt(long int x) {
+        
+        long int c=0,m;
```
Comment (SakshiShreya) [inline Sqrt/Implement Sqrt(x) without math library.txt]: Add space around operators.
Add space after comma.

File: Sqrt/Implement Sqrt(x) without math library.txt [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        for(long int i=1;i<=x;i++)
+        {
+            m=i*i;
+            if(m==x)
+            {
+               c=i;
+                break;
+            }
+            else if(m>x){
+
```
Comment (SakshiShreya) [inline Sqrt/Implement Sqrt(x) without math library.txt]: Remove the extra lines. You have not used new-line in `if` part of the block. Then maintain consistency in `else if` part also.

**Comment by jalajdorai** [comment]
@SakshiShreya I have made the changes please review this

**Comment by somya-kapoor** [comment]
@SakshiShreya Please review.

File: Sqrt/Implement Sqrt(x) without math library.cpp [hunk 4]
```diff
@@ -0,0 +1,42 @@
+/*Implement int sqrt(int x).
+Compute and return the square root of x, where x is guaranteed to be a non-negative integer.
+Since the return type is an integer, the decimal digits are truncated 
+and only the integer part of the result is returned.*/
+
+class Solution {
+public:
+    int mySqrt(long int x) {
+        long int c = 0, m;
+        for(long int i = 1;i <= x;i++)
```
Comment (raghav-dalmia) [inline Sqrt/Implement Sqrt(x) without math library.cpp]: add space after ;

**Comment by raghav-dalmia** [comment]
Also, Squash your commits.

**Comment by somya-kapoor** [comment]
@jalajdorai Are you working on it?

**Pull request closed without merging**